cards.tex

```diff
@@ -46,6 +46,22 @@
 \newcommand{\Reference}[2]{{\small \color{gray} \engschrift #1 #2}}
 \newcommand{\DamageType}[1]{\operatorname{\mbox{#1}}}
 
+\newcommand{\Tag}[2]{%
+    \tikz[baseline]{%
+        \node[anchor=base, text=white, fill=#1, font=\sffamily, text depth=.5mm] {#2};
+    }%
+}
+
+\definecolor{tagsBg}{RGB}{217, 196, 132}
+\definecolor{uncommonBg}{RGB}{152, 81, 61}
+\definecolor{rareBg}{RGB}{0, 38, 100}
+\definecolor{traitBg}{RGB}{94, 0, 0}
```

Comment:
I understand that using the same colors as in the books helps with parsing the information. I'm a bit conflicted though because I don't actually like the colors ...

Reply:
Yes, the color scheme is ... meh. 

I wanted to start somewhere, and until I have a way better understanding of whether the traits can be grouped or categorized in a meaningful way this is as good as everything else for now. 

We can easily change that later as well.